OpenRA.Game/OpenRA.Game.csproj

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Not sure what you mean here, if we merge this as-is then these changes will exist.